src/dtw.jl

```diff
@@ -12,6 +12,56 @@ function nanopore_dtw(x::Vector{Float32}, y::Vector{Float32},
     return dtw_bhattacharyya(kmerdists1, kmerdists2)
 end
 
+
+function multi_nanopore_dtw(x::Vector{Vector{Float32}},
+     kmerpath::String = "models/r9.4_70bps.u_to_t_rna.5mer.template.model")
+     kmers = loadkmers(kmerpath)
+     kmerdists::Array{Array{kmerdist}} = []
+     for i in x
+        push!(kmerdists, kmerdist_from_changepoints(i, detect_change_points(i, kmers)))
+     end
+     seqlengths::Array{Int64} = []
+     for s in kmerdists
+         push!(seqlengths, length(s))
+     end
```

Comment:
you can simply use broadcasting for stuff like this, just do 
seqlengths = length.(seqlengths) 

Pay attention to the dot there that means, do it for every element!